doc/news/changes/minor/20191217KonradSimon

```diff
@@ -0,0 +1,4 @@
+New: The TensorFunctionParser class allows to read 
+in a function similarly to the FunctionParser class using the MuParser.
+<br>
+(Timo Heister, 2019/11/04)
```

Comment:
I think you want to give yourself credit here :-) (Unless @konsim83 is really an alias for @tjhei , though I doubt that :-) )

Please also add a newline at the end of the file.

Reply:
Oops. Copy and paste is dangerous... :-) Corrected.